EduPerfTests/Program.cs

```diff
@@ -229,20 +303,20 @@ static RemoteWebDriver LaunchDriver(Browser browser)
                         Thread.Sleep(2000);
                         break;
                     default:
-                        throw new Exception(string.Format("Unexpected browser: {0}", browser));
+                        throw new Exception($"Unexpected browser: {browser}");
                 }
 
-                driver.Manage().Window.Maximize();
+                //driver.Manage().Window.Maximize();
```

Comment:
Good catch. I commented it out temporarily while testing because I was running half screen...